Articles/ShortArticles/DiscontinuingSoftwareProjects.md

```diff
@@ -0,0 +1,136 @@
+# Discontinuing a Research Software Project
+
+HPC/CSE software projects involve at least two kinds of *deep* know-how.
+One is the computational science the software aims to enable.
+The other is the computer science (aka software engineering) involved in designing, developing, deploying and supporting the software for users.
+
+Because the ultimate goal of most HPC/CSE projects is the advancement of the underlying science, publications concerning the computational science know-how are often the priroity.
+When such a project ends, there is little risk the computational science know-how will be lost.
```

Comment:
Great idea. I will add a paragraph specific to this topic

Reply:
Ok, I tried to address with the changes in cdd599a97b9825300dd3f0e207ee7b0d448e9c0e. Let me know what you think.

Reply:
@markcmiller86, I think you did a good job on addressing the issue of reproducibility.  But one has to ask if a research project should be funded in the first place if the sponsors don't care if the research has any chance of being reproduced.  But I guess that is a different battle 🤷

Reply:
@bartlettroscoe I do think you have raised an important question. That said, I think the responsibility for ensuring reproducibility falls less on the tool makers and more on the tool users. Certainly, tool makers can wind up engaging in practices that hinder reproducibility and we all agree such practices should be avoided.

But, assuming no project is intentionally doing the *wrong things* (as far as enabling reproducibility goes), I think the majority of responsibility for reproducibility falls on those who use the tools to draw conclusions or otherwise make the case about any specific results. They have the responsibility of ensuring others in the community have all the information they need to reproduce those results. IMHO, that means capturing whatever snapshots of the tool(s) they used *at the time* those results were generated and does not somehow mean they or even the tool makers have an obligation to *maintain* the tool(s) beyond that time.

But, that is more detail that I am prepared to get into here. So, I simply leave it as an open question. Ok?

Reply:
@markcmiller86, sounds good to me.